integration_tests/dbt_project.yml

```diff
@@ -1,10 +1,10 @@
 
 name: 'event_logging_integration_tests'
-version: '1.0'
+version: '0.3.0'
+config-version: 2
 
 profile: 'integration_tests'
 
-require-dbt-version: ">=0.15.0"
 
 models:
   pre-hook: "{{ logging.log_model_start_event() }}"
```

Comment:
These are running successfully. Eventually, I _think_ these will need to be `+pre-hook` and `+post-hook`, but there's an odd behavior around pre/post hook rendering right now (see https://github.com/fishtown-analytics/dbt/issues/2583). So I'm ok merging this as is.

Reply:
@clrcrl thoughts?

Reply:
@jtcohen6 the `+` is always optional, and is only necessary if a dictionary config is provided!

Reply:
This was news to me! Though I got pretty confused by this reply. I think you meant something more like:

> The `+` is optional _unless_ the config you are specifying has a dictionary of values, since dbt _could_ theoretically get confused as to whether this is part of a model path. Example:

```
models:
  +partition_by:
    name: id
    data_type: date
```